app/views/users/edit.html.erb

```diff
@@ -5,12 +5,13 @@
             <hr />
             <%= form_for @user do |f| %>
                 <%= render 'shared/error_messages', object: f.object %>
+                <div class="text-center">
                 <% if @user.picture? %>
                     <%= image_tag @user.picture.url, class: "br-max" %>
                 <% else%>
-                    <%= image_tag 'default.jpg' %>
+                    <%= image_tag 'default.jpg', class: "br-max" %>
                 <% end %>
-
+                </div>
                 <%= f.file_field :picture, autocomplete: "image", :multipart => true %>
 
```

Comment:
Remove ':multipart => true'...
We don't need to put this one if we are using form_for..

https://guides.rubyonrails.org/form_helpers.html#uploading-files